modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,22 @@
+== Inline formatting.
+
+List bullets and list numbers can now be inline-formatted.
+
+Inline-formatting, also known as spot-formatting, is the direct formatting of an object. It contrasts with formatting applied to an object because that object is a particular type of object.
+
+If an end-user selects the entire contents of a list item and applies inline formatting — such as a change of color or change of font-size — this spot-formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection take on the inline-formatting applied to the selection.
```

Comment:
Wondering if `'bullets or list numbers'` should instead be `'list bullets or numbers'` or `'list bullets or list numbers'`?

Reply:
Edited to go with the second suggestion — `list bullets or list numbers` — for clarity’s sake.